Resources/config/services.yml

```diff
@@ -16,6 +16,6 @@ services:
 
     umanit_block.form.panel_type:
         class: Umanit\BlockBundle\Form\PanelType
-        arguments: ['@umanit_block.resolver.block_manager_resolver', '@doctrine.orm.entity_manager']
+        arguments: ['@umanit_block.resolver.block_manager_resolver', '@doctrine.orm.entity_manager', '@translator']
```

Comment:
The `'@translator'` adds a dependency on a translation component. Could you make this reference optional by using `'@?translator'`?